spock-core/src/main/java/org/spockframework/runtime/model/IErrorContext.java

```diff
@@ -0,0 +1,14 @@
+package org.spockframework.runtime.model;
+
+import org.spockframework.util.Nullable;
+
+public interface IErrorContext {
```

Comment:
Do the user need to know that interface?
If yes, I would say Javadoc is missing.